bucket/toggl.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://www.toggl.com/",
+    "description": "Time Tracking for Tech Companies",
+    "version": "7.5.119",
+    "license": "Proprietary",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v7.5.119/TogglDesktopInstaller-x64-7.5.119.exe",
+            "hash": "E46C141EF0E214092E230E40DEEB8AE19C50BACEC15385772BBE679DE6A355E8"
+        }
+    },
+    "checkver": {
+        "url": "https://toggl.github.io/toggldesktop/",
+        "regex": "releases/download/v([\\d.]+)/TogglDesktopInstaller-x64-([\\d.]+)\\.exe"
+
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v$version/TogglDesktopInstaller-x64-$version.exe"
+            }
+        }
+    }
+}
```

Comment:
For the checkver I need to find better way

```suggestion
{
    "version": "7.5.162",
    "description": "Time Tracking for Tech Companies",
    "homepage": "https://toggl.com/toggl-desktop",
    "license": {
        "identifier": "Proprietary",
        "url": "https://www.toggl.com/legal"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v7.5.162/TogglDesktopInstaller-x64-7.5.162.exe#/dl.7z",
            "hash": "5967842808bb73a7705afdca6b5fd7553da5f4967b864e7e0f6d83525cfd9e13"
        },
        "32bit": {
            "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v7.5.162/TogglDesktopInstaller-7.5.162.exe#/dl.7z",
            "hash": "b1dcab85ebc2f96cb4f87a21f0819d053932e66894e6304bb7a1cca9c882f6d8"
        }
    },
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstal*\" -Recurse",
    "shortcuts": [
        [
            "TogglDesktop.exe",
            "Toggl Desktop"
        ]
    ],
    "checkver": {
        "url": "https://toggl-open-source.github.io/toggldesktop/assets/changelog/win.json",
        "regex": "\"version\":\"([\\d.]+)\""
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v$version/TogglDesktopInstaller-x64-$version.exe#/dl.7z"
            },
            "32bit": {
                "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v$version/TogglDesktopInstaller-$version.exe#/dl.7z"
            }
        }
    }
}

```

Reply:
Mate, what's with the attitude? I'm new and still trying to figure things out here (github and scoop both), there's no need to react quite as harshly. 

Thank you for making suggestions above, i'll check those out.

Reply:
```json
    "checkver": {
        "url": "https://toggl-open-source.github.io/toggldesktop/download/windows64-stable/",
        "regex": "x64-([\\d.]+)\\.exe"
    },
```

This works for me.